pkg/typeutil/json.go

```diff
@@ -15,6 +16,7 @@ func (j JSONDuration) MarshalJSON() ([]byte, error) {
 	return json.Marshal(j.Duration.String())
 }
 
+// Deprecated
```

Comment:
No replacements or so? I really like it when, just from the comment, I already know whats to use instead of this.

Reply:
It is only use by `buildutil`. So the replacement would be to move it into its own package. I am generally not against having it in the SDK, but I am against using it only in one place.